CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ cmake_minimum_required (VERSION 2.8.10)
 project (nix C CXX)
 
 set(VERSION_MAJOR 1)
-set(VERSION_MINOR 1)
+set(VERSION_MINOR 2)
 set(VERSION_PATCH 0)
 
```

Comment:
On master we should in the future probably bump the versions immediately after the release.